packages/formation/package.json

```diff
@@ -18,7 +18,8 @@
     "url": "git+https://github.com/department-of-veterans-affairs/veteran-facing-services-tools.git"
   },
   "scripts": {
-    "build": "node ./scripts/build.js"
+    "build": "node ./scripts/build.js",
+    "lint": "npx vagov-eslint lint:js"
```

Comment:
add the `lint` script which calls the `npx` command

Reply:
Do you need `npx` in a task in here? I think npm/yarn will work without it (similar to the tasks that run lerna).